docker/conda_build.dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG build_hdf5="NO"
+ARG pyne_test_base=openmc
+ARG ubuntu_version=22.04
+
+FROM ubuntu:${ubuntu_version} AS base_conda
+
+# Ubuntu Setup
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update \
+    && apt-get install -y --fix-missing \
+        wget \
+        bzip2 \
+        ca-certificates \
+    && apt-get clean -y
+
+RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
+    wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh
+
+ENV PATH /opt/conda/bin:$PATH
+
+RUN conda config --add channels conda-forge
+RUN conda update -n base -c defaults conda
+RUN conda install -y conda-libmamba-solver
+RUN conda config --set solver libmamba
+RUN conda install -y mamba
+RUN conda uninstall -y conda-libmamba-solver
+RUN conda config --set solver classic
+RUN conda update -y --all && \
+    mamba install -y \
+                cmake \
+                git \
+                setuptools \
+                pytest \
+                pytables \
+                jinja2 \
+                cython \
+                && \
+    mamba install -y --force-reinstall libsqlite && \
+    conda clean -y --all
+RUN mkdir -p `python -m site --user-site`
+
+ENV CC /opt/conda/bin/x86_64-conda_cos6-linux-gnu-gcc
+ENV CXX /opt/conda/bin/x86_64-conda_cos6-linux-gnu-g++
+ENV CPP /opt/conda/bin/x86_64-conda_cos6-linux-gnu-cpp
+
+# put conda on the path
+ENV LD_LIBRARY_PATH /opt/conda/lib:$LD_LIBRARY_PATH
+
+# make starting directory
+RUN mkdir -p $HOME/opt
+RUN echo "export PATH=$HOME/.local/bin:\$PATH" >> ~/.bashrc
+
+FROM base_conda AS moab
+RUN conda install conda-forge::moab
+
+FROM moab AS dagmc
+RUN conda install conda-forge::dagmc
+
+FROM dagmc AS openmc
+RUN conda install conda-forge::openmc
+
+# Build/Install PyNE from develop branch
+FROM ${pyne_test_base} AS pyne-dev
+
+RUN export PYNE_HDF5_ARGS="" ;\
+    && cd $HOME/opt \
+    && git clone -b develop --single-branch https://github.com/pyne/pyne.git \
```

Comment:
Please use COPY command here and copy the source to the dockerfile.

Reply:
To apply this change, I have removed the `pyne-dev` stage. Now, we will build the existing `pyne` stage in `docker_publish.yml` since that uses the COPY command.